src/eigen/lobpcg_hyper_impl.jl

```diff
@@ -56,12 +58,14 @@ boverlap(A, B) = A' * B
 
 # Perform a Rayleigh-Ritz for the N first eigenvectors.
 function RR(X, AX, BX, N)
-    F = eigen(Hermitian(boverlap(X, AX, assume_hermitian=true)))  # == Hermitian(X' AX)
-    F.vectors[:,1:N], F.values[1:N]
+    @timeit timer "Rayleigh-Ritz" begin
+        F = eigen(Hermitian(boverlap(X, AX, assume_hermitian=true)))  # == Hermitian(X' AX)
+        F.vectors[:,1:N], F.values[1:N]
+    end
 end
 
 import LinearAlgebra: cholesky
-function cholesky(X::Union{Matrix{ComplexF16}, Hermitian{ComplexF16,Matrix{ComplexF16}}})
+@timeit timer function cholesky(X::Union{Matrix{ComplexF16}, Hermitian{ComplexF16,Matrix{ComplexF16}}})
```

Comment:
Remove this. This is a super special function and we never time the other versions in LinearAlgebra.